src/components/Popover/Popover.js

```diff
@@ -46,12 +46,17 @@ class Popover extends Component {
     /**
      * alignment of the popover relative to the button
      */
-    align: alignPropType
```

Comment:
These changes to the Popover component should be in a separate PR  👀

Reply:
Yea, don't you worry. I'll cherry pick it away when we merge this. Having a hard time depending on two different `circuit-ui` branches in the dashboard :)

Reply:
Commit is dropped.